.azure-pipelines/default.yml

```diff
@@ -0,0 +1,28 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+     
+     mkdir dist
+    displayName: 'Prep Environment, Install Packages'
+
+  - script: 'python ./build_package.py --dest %BUILD_SOURCESDIRECTORY%\dist "azure-keyvault"'
```

Comment:
It would be a good idea to take this a step further and make azure-keyvault be a parameter that gets passed into the devops build. You can default the value to azure-keyvault for now but at least it starts to generalize this process.